Project.toml

```diff
@@ -17,6 +17,7 @@ JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
I think in the long term it would be nice to homogenize the different plotting libraries to reduce the number of dependencies. I do not know if we should choose `Plots` or `Gadfly`, however.

Reply:
we can add Plots for now and adapt / homogenize later

Reply:
Good take. I propose to discuss on the plotting library in a separate issue, and merge this PR with Plots.